tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1540,38 +1540,38 @@ public void TestPrefixAndSuffix()
 
             DataFrame prefix = df.AddPrefix("Prefix_");
             IEnumerable<DataViewSchema.Column> prefixNames = ((IDataView)prefix).Schema;
-            foreach ((DataViewSchema.Column First, DataViewSchema.Column Second) in columnNames.Zip(((IDataView)df).Schema))
+            foreach ((DataViewSchema.Column First, DataViewSchema.Column Second) in columnNames.Zip(((IDataView)df).Schema, (e1, e2) => (e1, e2)))
```

Comment:
What does the `(e1, e2)` delegate do? I don't see it being used anywhere

Reply:
It is taking in 2 parameters, `e1` and `e2`, and then returning the two parameters as a Tuple `(e1, e2)`. This is what it was doing before, except the `Zip` overload that doesn't take a selector and returns a Tuple is only in .NET Core 3.0.

https://apisof.net/catalog/System.Linq.Enumerable.Zip%3CTFirst,TSecond%3E(IEnumerable%3CTFirst%3E,IEnumerable%3CTSecond%3E)